Frameworks/Foundation/NSMutableOrderedSet.mm

```diff
@@ -277,4 +285,11 @@ - (void)unionSet:(NSSet*)other {
     UNIMPLEMENTED();
 }
 
+/**
+ @Status Interoperable
+*/
+- (id)copyWithZone:(NSZone*)zone {
+    return [[NSMutableOrderedSet alloc] initWithOrderedSet:self];
```

Comment:
I disagree here. copyWithZone should return an instance of NSMutableOrderedSet (since that is the class being implemented here) but it should be a new, copied instance ... 

---
In reply to: [63976700](https://github.com/Microsoft/WinObjC/pull/482#discussion_r63976700) [](ancestors = 63976700)